x/simulation/account.go

```diff
@@ -45,3 +46,45 @@ func RandomAccounts(r *rand.Rand, n int) []Account {
 
 	return accs
 }
+
+// FindAccount iterates over all the simulation accounts to find the one that matches
+// the given address
+func FindAccount(accs []Account, address sdk.AccAddress) (Account, bool) {
+	for _, acc := range accs {
+		if acc.Address.Equals(address) {
+			return acc, true
+		}
+	}
+
+	return Account{}, false
```

Comment:
```suggestion
	return nil, false
```

Reply:
not possible, unfortunately, unless I define named return variables

Reply:
why? Isn't an `Account` an interface?